### ISSUE TITLE
Crash when trying to teleport to Toon Escape

### ISSUE THREAD

**Issue by TheAfroOfDoom** [open-toontown/open-toontown#68]
**How to reproduce:**
run the following spellbook command:
`mg teleport 2d`

**Expected behavior:**
the [Toon Escape](https://toontown.fandom.com/wiki/Toon_Escape) trolley game begins

**Actual behavior:**
the game client crashes with the following traceback:

```
Traceback (most recent call last):
  File "open-toontown\otp\launcher\LauncherBase.py", line 150, in mainLoop
    self._runTaskManager()
  File "open-toontown\otp\launcher\LauncherBase.py", line 130, in _runTaskManager
    taskMgr.run()
  File "Panda3D-1.11.0-x64\direct\task\Task.py", line 546, in run
    self.step()
  File "Panda3D-1.11.0-x64\direct\task\Task.py", line 497, in step
    self.mgr.poll()
  File "Panda3D-1.11.0-x64\direct\distributed\ConnectionRepository.py", line 613, in readerPollUntilEmpty
    while self.readerPollOnce():
  File "Panda3D-1.11.0-x64\direct\distributed\ConnectionRepository.py", line 620, in readerPollOnce
    self.handleDatagram(self.private__di)
  File "open-toontown\otp\distributed\OTPClientRepository.py", line 2120, in handleDatagram
    self.handler(msgType, di)
  File "open-toontown\otp\distributed\OTPClientRepository.py", line 1674, in handlePlayGame
    self.handleMessageType(msgType, di)
  File "open-toontown\otp\distributed\OTPClientRepository.py", line 2003, in handleMessageType
    self.gotInterestDoneMessage(di)
  File "open-toontown\otp\distributed\OTPClientRepository.py", line 2039, in gotInterestDoneMessage
    self.__playBackGenerates(handle)
  File "open-toontown\otp\distributed\OTPClientRepository.py", line 2312, in __playBackGenerates
    self.__generateObject(doId, parentId, zoneId, classId, di, other)
  File "open-toontown\otp\distributed\OTPClientRepository.py", line 2379, in __generateObject
    distObj = self.generateWithRequiredFields(dclass, doId, di, parentId, zoneId)
  File "Panda3D-1.11.0-x64\direct\distributed\ClientRepositoryBase.py", line 268, in generateWithRequiredFields
    distObj.updateRequiredFields(dclass, di)
  File "Panda3D-1.11.0-x64\direct\distributed\DistributedObject.py", line 343, in updateRequiredFields
    self.announceGenerate()
  File "open-toontown\toontown\minigame\DistributedMinigame.py", line 106, in announceGenerate
    self.load()
  File "open-toontown\toontown\minigame\DistributedTwoDGame.py", line 56, in load
    self.assetMgr = ToonBlitzAssetMgr.ToonBlitzAssetMgr(self)
  File "open-toontown\toontown\minigame\ToonBlitzAssetMgr.py", line 14, in __init__
    self.load()
  File "open-toontown\toontown\minigame\ToonBlitzAssetMgr.py", line 71, in load
    for n in range(ToonBlitzGlobals.NumSections[self.game.getSafezoneId()] + 1):
KeyError: 22000
```

**Comment by LittleToonCat** [comment]
Using the Minigame Magic Word in Welcome Valley should be fixed as of 5151d94ac438520ab4fa589832270395126d906e.

Make sure that your game has the latest changes and try again.

**Comment by TheAfroOfDoom** [comment]
pulling latest changes fixes, thanks
